README.md

```diff
@@ -0,0 +1,84 @@
+# JsonApiDotNetCore MongoDB Repository
+
+[![NuGet version][nuget-image]][nuget-url]
+[![Downloads][downloads-image]][nuget-url]
+[![Build Status](https://travis-ci.com/mrnkr/JsonApiDotNetCore.MongoDb.svg?branch=master)](https://travis-ci.com/mrnkr/JsonApiDotNetCore.MongoDb)
+[![codecov](https://codecov.io/gh/mrnkr/JsonApiDotNetCore.MongoDb/branch/master/graph/badge.svg)](https://codecov.io/gh/mrnkr/JsonApiDotNetCore.MongoDb)
+[![license][license]](https://github.com/mrnkr/JsonApiDotNetCore.MongoDb/blob/master/LICENSE)
+
+[nuget-image]:https://img.shields.io/nuget/v/JsonApiDotNetCore.MongoDb
+[nuget-url]:https://www.nuget.org/packages/JsonApiDotNetCore.MongoDb
+[downloads-image]:https://img.shields.io/nuget/dt/JsonApiDotNetCore.MongoDb
+[license]:https://img.shields.io/github/license/mrnkr/JsonApiDotNetCore.MongoDb
+
+Plug-n-play implementation of `IResourceRepository<TResource, TId>` allowing you to use MongoDb with your `JsonApiDotNetCore` APIs.
+
+## Installation and Usage
```

Comment:
I've cloned the repo and tried to run tests. They fail with message:
```
System.TimeoutException
A timeout occured after 30000ms selecting a server using CompositeServerSelector{ Selectors = MongoDB.Driver.MongoClient+AreSessionsSupportedServerSelector, LatencyLimitingServerSelector{ AllowedLatencyRange = 00:00:00.0150000 } }. Client view of cluster state is { ClusterId : "1", ConnectionMode : "Automatic", Type : "Unknown", State : "Disconnected", Servers : [{ ServerId: "{ ClusterId : 1, EndPoint : "Unspecified/localhost:27017" }", EndPoint: "Unspecified/localhost:27017", ReasonChanged: "Heartbeat", State: "Disconnected", ServerVersion: , TopologyVersion: , Type: "Unknown", HeartbeatException: "MongoDB.Driver.MongoConnectionException: An exception occurred while opening a connection to the server.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): No connection could be made because the target machine actively refused it. [::1]:27017
```

This would be the place to describe what needs to be installed. Is there some installer or a docker image that can be used? I'm assuming travisci has mongodb pre-installed?

Reply:
I've only just fixed the tests after migrating to the latest version of JADNC and I can confirm all that's necessary is to have a running instance of mongodb on `localhost:27017` with no authentication required. I am personally rocking a docker image for this - `mongo:latest`.

I'm planning on doing as @alastairtree suggested anyway and use mongo2go on my tests so that this is not necessary anymore.

Reply:
The point is that I have zero experience with mongodb, so 'just have a runnning instance' does not tell me what to do to make that happen. A docker cmdline or choco install command or something similar would help (see [here](https://github.com/json-api-dotnet/JsonApiDotNetCore#testing) for example).